sourmash/kmer_min_hash.hh

```diff
@@ -174,7 +174,14 @@ public:
         unsigned int dna_size = (dna.size() / 3) * 3; // floor it
         for (unsigned int j = 0; j < dna_size; j += 3) {
             std::string codon = dna.substr(j, 3);
-            aa += (_codon_table)[codon];
+            std::map<std::string,std::string>::iterator translated = _codon_table.find(codon);
```

Comment:
quick comment: you can replace `std::map<std::string,std::string>::iterator` with `auto` and avoid having to figure out the proper type (yay C++11)